code/modules/clothing/under/jobs/cargo.dm

```diff
@@ -15,6 +15,15 @@
 	can_adjust = FALSE
 	fitted = FEMALE_UNIFORM_TOP
 
+/obj/item/clothing/under/rank/cargo/qm/turtleneck
```

Comment:
Curious as to why this is an undersuit when turtle necks would generally be in the same layer as a jacket?

Reply:
The research director’s turtleneck is defined as an undersuit aswell, as why this one is defined as an undersuit? 

No idea, you’d have to contact the person who made the original port to BeeStation

Reply:
I'm guessing these people don't know how to dress themselves.

But yes, I was aware of the research director also being special.